src/scheduler/job_info.c

```diff
@@ -3198,8 +3198,8 @@ find_jobs_to_preempt(status *policy, resource_resv *hjob, server_info *sinfo, in
 		return NULL;
 	}
 	npolicy = nsinfo->policy;
-	njob = find_resource_resv_by_indrank(nsinfo->jobs, hjob->rank, hjob->resresv_ind);
-	prev_prio = njob->job->preempt;
+	hjob_dup = find_resource_resv_by_indrank(nsinfo->jobs, hjob->rank, hjob->resresv_ind);
```

Comment:
The entire sinfo has been dup'd.  I'd rather not call this hjob_dup.  Maybe hjob?  The first thing I thought of was this was a dup of job in the dup'd server.

Reply:
There is already a variable named hjob in use. It is technically a duplicated high priority job from the duplicate universe. If you don't like it, I can use hjob_new as Ravi suggested. I thought there wasn't anything new with the job that is why choose hjob_dup instead.